GrocifyApp.UI/src/components/Products/ProductMeasuresSelector.jsx

```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react';
```

Comment:
please add "query[key] !== undefined" to useApiRequest.makeRequest

 const queryString = Object.keys(query)
     .filter(key => query[key] !== null && query[key] !== "" **&& query[key] !== undefined**)
     .map(key => `${encodeURIComponent(key)}=${encodeURIComponent(query[key])}`)
     .join('&');